cluster/log-dump/log-dump.sh

```diff
@@ -438,6 +439,10 @@ function dump_nodes() {
     if [[ proc -eq 0 ]]; then
       proc=${max_dump_processes}
       wait
+      now="$(date +%s)"
+      if [[ ! -z "${LOG_DUMP_SSH_TIMEOUT}" && $((now - start)) -gt ${LOG_DUMP_SSH_TIMEOUT} ]]; then
```

Comment:
LOG_DUMP_SSH_TIMEOUT has to have defaulting - otherwise you risk failures if it will be unset.

Probably something like:
log_dump_ssh_timeout="${LOG_DUMP_SSH_TIMEOUT:=<some default value>}"
and here refer to log_dump_ssh_timeout


Another point is that we should make it explicit that it's a rough estimation of timeout, because wait may take time, so this timeout may be visibly exceeded..

Reply:
Fixed. Thanks for pointing out how to manage unset bash variables in such situations.